packages/dappmanager/src/initializeDb.ts

```diff
@@ -74,13 +73,7 @@ export default async function initializeDb(): Promise<void> {
   //   and the external IP from UPnP command succeeded
   const upnpAvailable =
     Boolean(publicIp && externalIp && internalIp !== publicIp) &&
```

Comment:
Do we use the `externalIp` for something? What does it even represent?

Reply:
we assume that when both IPs exist and are different from each other the DAppNode will not have a gateway with UPnP between itself and the internet